tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_joint
    joint_speed: 0.5
    motor:
        id: 1
        init: 0
        min: 0
        max: 4095
shoulder_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_joint
    joint_speed: 0.5
    motor:
        id: 2
        init: 0
        min: 0
        max: 4095
elbow_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow_joint
    joint_speed: 0.5
    motor:
        id: 3
        init: 0
        min: 0
        max: 4095
wrist_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_joint
    joint_speed: 0.5
    motor:
        id: 4
        init: 0
        min: 0
        max: 4095
hand_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: hand_joint
    joint_speed: 0.5
    motor:
        id: 5
        init: 0
        min: 0
        max: 4095